services/core-api/app/api/mines/response_models.py

```diff
@@ -714,3 +728,13 @@ def format(self, value):
         'description': fields.String,
         'active_ind': fields.Boolean
     })
+
+MINE_NOD_MODEL = api.model('Nod', {
+    'nod_guid': fields.String,
+    'nod_title': fields.String,
+    'permit': fields.Nested(PERMIT_BASE_MODEL)
+})
+
+PERMIT_NOD_MODEL = api.model('Nod', {'nod_guid': fields.String, 'nod_title': fields.String})
```

Comment:
are we missing permit_guid?

Reply:
oh, it is intentional. this is the response model of nods by permit. So, having permit_guid is un-necessary